### DISCUSSION TITLE
Support fot Tuya TH05

### DISCUSSION THREAD

**Discussion by Monacoslo** [pvvx/ZigbeeTLc#21] [Ideas]
Cheap Tuya TH05 modules look the same and are constantly on promotion on AliExpress. Please make support for them too.

**Comment by pvvx** [comment]
Contact those who work with PHY chips.

https://pvvx.github.io/TH-05/  - SoC: [PHY6222QC-W04I](http://en.phyplusinc.com/index.php/hardware_p?id=385) - Doesn't have an SDK for GCC. Only commercial version of the compiler.
They have high battery consumption.

Unsolved problems:
https://github.com/custom-components/ble_monitor/issues/1063
